scylla.go

```diff
@@ -181,20 +177,19 @@ func (p *scyllaConnPicker) shardOf(token murmur3Token) int {
 func (p *scyllaConnPicker) Put(conn *Conn) {
 	const maxExcessConnsFactor = 10
 
-	s := parseSupported(conn.supported)
-	if s.nrShards == 0 {
+	if conn.scyllaSupported.nrShards == 0 {
```

Comment:
Maybe load `nrShards` into a local var?

Reply:
Yes, you are right, it would save some typing. I'll adjust.